### DISCUSSION TITLE
Welcome to Tafe230231 Discussions!

### DISCUSSION THREAD

**Discussion by Victor-tafe** [Victor-tafe/Tafe230231#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by StephenHillenbrand** [comment]
Great, thanks Victor, I have a few questions I  will ask in other posts.

**Comment by StephenHillenbrand** [comment]
Can I have a list of who is completing the different branches?

**Comment by Victor-tafe** [comment]
Hey Stephen, I have been assigned to do the mortgage calculator feature branch and I believe Enver was assigned the currency converter feature branch.
________________________________
